postmaster/utils.py

```diff
@@ -254,6 +256,8 @@ def add_ldap_user_to_db(username, display_name):
 def validate_wtforms_password(form, field):
         """ Validates the password from a wtforms object
         """
+        # Prevent circular import on json_logger by importing here
+        from postmaster.ad import AD, ADException
```

Comment:
We may run into this in the future. It may be better to separate the logger out into its own logger.py file.